packages/react-cosmos-voyager/scripts/generate-use-cases.js

```diff
@@ -62,6 +62,16 @@ const useCases = {
     pkgs/nested/Baz/src/index.jsx
     pkgs/nested/Baz/fixtures/blank.js
   `),
+  'fixtures-dir-setting': (`
+    components/Foo.js
+    components/nested/Bar.jsx
+    components/nested/Baz.jsx
+    components/__fixtures__/Foo/blank.js
```

Comment:
Right, should be fixed now. I changed generated folder and forgot to update here.